wp_mobile_cache/src/repository/posts.rs

```diff
@@ -391,7 +1020,8 @@ mod tests {
         let retrieved = test_ctx
             .post_repo
             .select_by_rowid(&test_ctx.conn, &test_ctx.site, rowid)
-            .unwrap();
+            .unwrap()
```

Comment:
In some of these we have `unwrap().expect()` and in others we have `expect().expect()`

The latter is probably better in tests but WDYT?